dev/src/KeyPairGenerateTrait.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Dev;
+
+trait KeyPairGenerateTrait
```

Comment:
Yep, it's used in `Google\Cloud\Tests\Snippets\Storage\StorageObjectTest` as well. I put it in Dev because we don't have a folder in tests which is autoloaded and not specific to a test suite.